### ISSUE TITLE
Calculate optimal Faction event shuttle crew

### ISSUE THREAD

**Issue by pim69** [IAmPicard/StarTrekTimelinesSpreadsheet#188]
**Is your feature request related to a problem? Please describe.**
When competing in a faction event, on the first day as part of my quick start I'll note all the 3 slot shuttles in each faction, then try to build an optimal combination of crew in shuttle slots for maximum success % shuttle odds.  This is typically not exclusively 3-slot shuttles since I may not have crew with optimal skill combos for the 3-slot shuttles, or the bonus crew I've got don't fit into those slots optimally.

**Describe the solution you'd like**
Add a "Faction event" tab to the tool, where you can open ALL the shuttle missions available for the event, then have the tool recommend which shuttles to select and which crew give optimal success rates.  I don't know enough about math to know if this means 4 x 80% shuttles give better odds than 1x90%, 2x80%, and 1x70% shuttle success rate, but hopefully those involved in this project are better at math than I am!

Also, have a function to DESELECT one or more factions, if you are not looking for maximum VP ranking, and would rather focus on one or two factions for the event.

**Describe alternatives you've considered**
I don't know of any alternative tool or solution that could help me with this currently, I manually select crew in a bunch of slots and shuttles and do my best to get the best success % possible to plan running those same shuttles and crew for the entire event.

**Comment by Kamandi2** [comment]
I really like this idea.  I just commented on another post suggesting something almost exactly like this.  I can't imagine it would be something that would be done though because the code would have to be a nightmare.

**Comment by pim69** [comment]
yeah i actually saw 2 similar requests after mine that were very similar
but more detailed asking for more features.  one of them has an actual
spreadsheet where he's developed an algorithm that could potentially be
used, but they need someone to code it in javascript for the tool.  let's
hope someone who knows how to code it agrees with us it's a good idea!
